moveit_ros/perception/lazy_free_space_updater/CMakeLists.txt

```diff
@@ -8,5 +8,5 @@ target_link_libraries(${MOVEIT_LIB_NAME} moveit_occupancy_map_monitor ${catkin_L
 
 add_dependencies(${MOVEIT_LIB_NAME} ${sensor_msgs_EXPORTED_TARGETS})
 
-install(TARGETS ${MOVEIT_LIB_NAME} LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
-install(DIRECTORY include/ DESTINATION include)
+install(TARGETS ${MOVEIT_LIB_NAME} LIBRARY DESTINATION ${CATKIN_GLOBAL_LIB_DESTINATION})
```

Comment:
This installs to `GLOBAL` instead of `PACKAGE` now.
In other packages, e.g. `depth_image_octomap_updater` right above, it still installs to `CATKIN_PACKAGE_LIB_DESTINATION`.

I believe this is not intended to differ?

Reply:
Indeed, my bad. I will fix that.